src/main/scala/mesosphere/marathon/core/event/impl/stream/HttpEventStreamServlet.scala

```diff
@@ -24,13 +24,21 @@ class HttpEventSSEHandle(request: HttpServletRequest, emitter: Emitter) extends
 
   lazy val id: String = UUID.randomUUID().toString
 
+  val subscribedEventTypes = request.getParameterMap.getOrDefault("event_type", Array.empty).toSet
```

Comment:
`val subscribedEventTypes: Set[String] = request.getParameterMap.getOrDefault("event_type", Set.empty)(collection.breakOut)` might work if you have `import mesosphere.marathon.stream._`